connector_ebisumart/models/ebisumart_backend.py

```diff
@@ -154,24 +154,36 @@ def add_checkpoint(self, record):
 
     @api.multi
     def _import_orders(self, model):
+        description = (
+            f'JOB: Prepare the import of records modified in Ebisumart '
+            f'({self.env[model]._description})'
+        )
```

Comment:
As I always say, don't repeat the same code multiple times unless there is a good reason to do so. Cut this out into a method.